4.figures/main_figure_5/scripts/3.main_figure_5.r

```diff
@@ -10,118 +10,121 @@ output_main_figure_5 <- file.path(
+img_width <- ncol(second_top_feat_img_max)
 
 # Calculate the aspect ratio
 aspect_ratio <- img_height / img_width
 
 # Plot the image montage to a ggplot object
-radial_montage_Null_max <- ggplot() +
+second_top_montage_max <- ggplot() +
   annotation_custom(
-    rasterGrob(radial_feat_img_max, interpolate = TRUE),
+    rasterGrob(second_top_feat_img_max, interpolate = TRUE),
     xmin = -Inf, xmax = Inf, ymin = -Inf, ymax = Inf
   ) +
   theme_void() +
   coord_fixed(ratio = aspect_ratio, clip = "off") +
-  theme(plot.margin = margin(0, 0, 0, 0, "cm"))  # Adjust margins as needed
+  theme(plot.margin = margin(0, 0, 0, 0, "cm"))  # Set margins to zero
+second_top_montage_max
 
-radial_montage_Null_max
 
-radial_feat_path_min = file.path("./radial_feature_montage_min.png")
-radial_feat_img_min = png::readPNG(radial_feat_path_min)
+second_top_feat_path_min = file.path("./mito_radial_feature_montage_min.png")
+second_top_feat_img_min = png::readPNG(second_top_feat_path_min)
 
 # Get the dimensions of the image
-img_height <- nrow(radial_feat_img_min)
-img_width <- ncol(radial_feat_img_min)
+img_height <- nrow(second_top_feat_img_min)
+img_width <- ncol(second_top_feat_img_min)
 
 # Calculate the aspect ratio
 aspect_ratio <- img_height / img_width
 
 # Plot the image montage to a ggplot object
-radial_montage_WT_min <- ggplot() +
+second_top_montage_min <- ggplot() +
   annotation_custom(
-    rasterGrob(radial_feat_img_min, interpolate = TRUE),
+    rasterGrob(second_top_feat_img_min, interpolate = TRUE),
     xmin = -Inf, xmax = Inf, ymin = -Inf, ymax = Inf
   ) +
   theme_void() +
   coord_fixed(ratio = aspect_ratio, clip = "off") +
   theme(plot.margin = margin(0, 0, 0, 0, "cm"))  # Adjust margins as needed
 
-radial_montage_WT_min
+second_top_montage_min
+
+top_row_montage <- (
+   free(top_feat_montage_max) +
+   second_top_montage_max
+) + plot_layout(widths = c(1,1), guides = "collect")
 
-top_montage <- (
-   free(radial_montage_Null_max) | 
-   corr_montage_Null_min
-) + plot_layout(widths = c(1,1))
+top_row_montage
 
-top_montage
 
-bottom_montage <- (
-   free(radial_montage_WT_min) | 
-   corr_montage_WT_max
-) + plot_layout(widths = c(1,1))
+bottom_row_montage <- (
+   top_feat_montage_min +
+   second_top_montage_min
+) + plot_layout(widths = c(1,1), guides = "collect")
 
-bottom_montage
+bottom_row_montage
 
 align_plot <- (
-    top_montage /
-    bottom_montage
-) + plot_layout(heights = c(1,1))
+    top_row_montage /
+    bottom_row_montage
+) + plot_layout(heights = c(1.0, 1.0))
 
 align_plot
 
 fig_5_gg <- (
   align_plot
-) + plot_annotation(tag_levels = list(c("A", "B", "", ""))) & theme(plot.tag = element_text(size = 28))
+) + plot_annotation(tag_levels = list(c("A", "B", "", ""))) & theme(plot.tag = element_text(size = 30))
 
 # Save or display the plot
-ggsave(output_main_figure_5, plot = fig_5_gg, dpi = 500, height = 14, width = 18)
+ggsave(output_main_figure_5, plot = fig_5_gg, dpi = 500, height = 16, width = 22)
```

Comment:
I'm not sure where to make the suggestion to actually impact the code, but this figure montage should include the cell line code as well (to keep consistent)

Reply:
Ahhh thanks for catching! Let me add it back in now!

Reply:
Added!